Cargo.toml

```diff
@@ -60,3 +60,6 @@ time = ">=0.1"
 [build-dependencies]
 syntex = { version = "0.31.*", optional = true }
 pnet_macros = { path = "pnet_macros", version = ">=0.6" }
+
+[dev-dependencies.pcapng]
+git = "https://github.com/kornholi/pcapng.git"
```

Comment:
I've been doing some reading, and it seems that packages on crates.io cannot have dependencies on git repositories.

Either the author will need to publish their work to crates.io, or we can't have this dependency.